controllers/overcloudipset_controller.go

```diff
@@ -96,7 +96,7 @@ func (r *OvercloudIPSetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 
 	// iterate over the requested hostCount
 	for count := 0; count < instance.Spec.HostCount; count++ {
-		hostname := fmt.Sprintf("%s-%d", instance.Name, count)
+		hostname := fmt.Sprintf("%s-%d", instance.Spec.Role, count)
```

Comment:
@stuggi @abays This makes it so we use the Role as the basis for the hostnames.

Reply:
yes, and we should make sure the role gets converted to lowercase as the role is case sensitive to construct the correct HostNameFormat for the deployment. I'll do that in my patch